src/styles/app.less

```diff
@@ -14,12 +14,12 @@ body {
   padding-top: 1rem;
 }
 
-@btn-size: 60px;
+@btn-size: 80px;
```

Comment:
This is a pretty big bump - why 80px?

Reply:
We wanted a larger hitbox for mobile device users. The buttons were a bit too small for touch screens.

Reply:
Awesome, go ahead!

Reply:
What do you think James?

###### <a href="https://getunblocked.com"><img src="https://avatars.githubusercontent.com/in/166219" width=20 align="center" /></a>&nbsp;&nbsp;Comment posted from [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/c9e2a4b2-01a5-47a9-80a8-37e071edc954?message=a7eff2c3-9cd7-463d-a1ce-d7fbabcc258c).

Reply:
Looks good!